### DISCUSSION TITLE
Mining Modus Überlappende Texte Version 3.23.1a LIVE.9204897

### DISCUSSION THREAD

**Discussion by BTSVDanielk** [rjcncpt/StarCitizen-Deutsch-INI#557] [Übersetzungsvorschläge]
Guten Abend.

Im mining Modus sind sich 2 Wörter im Weg. Siehe Foto. Ich habe leider noch kein neuen Vorschlag.

Version
3.23.1a LIVE.9204897 14.06.2024 - 02:18 Uhr

![ScreenShot-2024-06-23_16-49-22-B99](https://github.com/rjcncpt/StarCitizen-Deutsch-INI/assets/167604723/58526f50-7b5f-4415-b3a9-05ce30f2cf5c)

Gruß Daniel

**Comment by MaxM1211** [comment]
Huhu, mein Vorschlag wäre die Zielmodi im Englischen zu lassen (LOCK,AUTO,...) Dabei sollte dann `Erfassung` wieder zu `Lock` werden
@rjcncpt Hast du noch eine andere Idee?

**Reply by rjcncpt** [reply]
Ich hätte fast gesagt wir lassen das weil ich glaube das die UIs sowieso irgendwann bis 4.0 angepasst werden. Aber ja, wäre auch eine Möglichkeit. Ich bin unentschlossen

**Reply by BTSVDanielk** [reply]
In der Vulture habe ich es eben auch entdeckt. :)

**Reply by BTSVDanielk** [reply]
die gleiche Überlappung vom Text.
